### DISCUSSION TITLE
Windows 10 support status

### DISCUSSION THREAD

**Discussion by ofer-dev** [microsoft/win32-app-isolation#41] [Q&A] [answered]
Will Windows 10 be supported? According to https://github.com/microsoft/win32-app-isolation/blob/main/relnotes/windows-release-notes.md at the moment only windows 11 is supported.

**Comment by freewong** [comment]
please support windows 10,please.

**Comment by cchavez-msft** [comment, marked as answer]
Hi, @ofer-dev and @freewong. AppSilo applications running in Windows 10 will run at FullTrust. You can see this in our release official documentation: https://learn.microsoft.com/en-us/windows/win32/secauthz/app-isolation-release-notes
